Packs/CrowdStrikeFalconSandbox/Integrations/CrowdstrikeFalconSandboxV2/CrowdstrikeFalconSandboxV2.py

```diff
@@ -0,0 +1,532 @@
+
+    }
+    return tableToMarkdown('Scan Results:', scan_response, headers=list(table_field_dict.keys()),
+                           headerTransform=lambda x: table_field_dict.get(x, x), removeNull=True)
+
+
+def get_dbot_score(filehash, threat_score: int):
+    def calc_score():
+        return {3: 0,
+                2: 3,
+                1: 2,
+                0: 1}.get(threat_score, 0)
+
+    return Common.DBotScore(indicator=filehash, integration_name='CrowdStrike Falcon Sandbox V2',
+                            indicator_type=DBotScoreType.FILE, score=calc_score(),
+                            malicious_description=f'Score of {calc_score()}',
+                            reliability=DBotScoreReliability.get_dbot_score_reliability_from_str(
+                                INTEGRATION_RELIABILITY))
+
+
+def get_submission_arguments(args: Dict[str, Any]) -> Dict[str, Any]:
+    return {camel_case_to_underscore(arg): args[arg] for arg in SUBMISSION_PARAMETERS if args.get(arg)}
+
+
+def submission_response(client, response, polling):
+    submission_res = [CommandResults(outputs_prefix='CrowdStrike.Submit', outputs_key_field='submission_id',
+                                     raw_response=response, outputs=response,
+                                     readable_output=tableToMarkdown("Submission Data:", response,
+                                                                     headerTransform=underscore_to_space)),
+                      CommandResults('CrowdStrike.JobID', outputs=response['job_id'],
+                                     readable_output=f"JobID: {response['job_id']}"),
+                      CommandResults('CrowdStrike.EnvironmentID', outputs=response['environment_id'],
+                                     readable_output=f"EnvironmentID: {response['environment_id']}")]
+
+    if not polling:
+        return submission_res
+    else:
+        return_results(submission_res)  # return early
+    return crowdstrike_scan_command(client, {'file': response['sha256'], 'JobID': response['job_id'],
+                                             "Polling": True})
+
+
+def crowdstrike_submit_url_command(client: Client, args: Dict[str, Any]):
+    submission_args = get_submission_arguments(args)
+    url = args['url']
+    response = client.submit_url(url, submission_args)
+    return submission_response(client, response, args.get('Polling'))
+
+
+def crowdstrike_submit_sample_command(client: Client, args: Dict[str, Any]):
+    file_contents = demisto.getFilePath(args['entryId'])
+    submission_args = get_submission_arguments(args)
+    response = client.submit_file(file_contents, submission_args)
+    return submission_response(client, response, args.get('Polling'))
+
+
+def crowdstrike_analysis_overview_command(client: Client, args: Dict[str, Any]):
+    result = client.analysis_overview(args['file'])
+    file = Common.File(Common.DBotScore.NONE, sha256=result['sha256'], size=result['size'], file_type=result['type'],
```

Comment:
does this return dbot score for you?

Reply:
from what I can see only scan should return dbot score. The api has 2 different fields, threat_score and threat_level. Threat_level only gets returned for scan, and we dont relate to threat_score

Reply:
lets discuss